cmd/manager/main.go

```diff
@@ -510,7 +511,7 @@ func startOperator(stopChan <-chan struct{}) error {
 	go asyncTasks(mgr, cfg, managedNamespaces, operatorNamespace, string(operatorInfo.OperatorUUID))
 
 	log.Info("Starting the manager", "uuid", operatorInfo.OperatorUUID,
-		"namespace", operatorNamespace, "version", operatorInfo.BuildInfo.Version,
+		"namespace", operatorNamespace, "esversion", operatorInfo.BuildInfo.Version,
```

Comment:
Is this intentional?

```suggestion
		"namespace", operatorNamespace, "version", operatorInfo.BuildInfo.Version,
```

Reply:
Overagressive IntelliJ refactoring :-(